static/loader.js

```diff
@@ -191,7 +191,8 @@
             if ('tvp_sidebar:video_click' === eventName) {
               var data = e.data;
               var selectedVideo = data.selectedVideo || {};
-              var runTime = (data.runTime || (isset(__TVPage__) ? __TVPage__ : {}) ).config[id];
+              var dataVideos = data.videos;
```

Comment:
What if videos is not defined? you can do something like:

`var dataVideos = isset(data,'videos') ? data.videos : [];`

Then later on you won't need the `if (dataVideos) {` cause for loops don't brake with empty arrays.